index.html

```diff
@@ -4,29 +4,48 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-
-    <meta property="og:url" content="https://bellshade.github.io/" />
-    <meta property="og:type" content="website" />
-    <meta property="og:title" content="Bellshade - Learn with Our Community" />
-    <meta property="og:description" content="Komunitas untuk mengelola dan menulis kode untuk proyek pendidikan sumber terbuka Indonesia" />
-    <meta property="og:image" content="https://raw.githubusercontent.com/reacto11mecha/bellshade.github.io/gh-pages/og_img.png" />
-
+    <link rel="icon" type="image/x-icon" href="/favicon.ico" />
+    <title>Bellshade - Learn with Our Community</title>
+    <link rel="dns-prefetch" href="https://bellshade-server.herokuapp.com" />
     <link rel="preconnect" href="https://fonts.googleapis.com" />
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
 
-    <link rel="shortcut icon" href="/favicon.ico" />
+    <meta name="title" content="Bellshade - Learn with Our Community" />
+    <meta name="description" content="Komunitas untuk mengelola dan menulis kode untuk proyek pendidikan sumber terbuka Indonesia." />
+    <meta name="keywords" content="bellshade, komunitas, wpu, web programming unpas, community, programming, unpas" />
+    <meta name="robots" content="index, follow" />
+    <meta name="googlebot" content="index, follow" />
+    <meta name="msnbot" content="index, follow" />
+    <meta name="yahoobot" content="index, follow" />
+    <meta name="bingbot" content="index, follow" />
+    <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+    <meta name="author" content="Bellshade" />
+    <meta name="copyright" content="Bellshade" />
+    <meta name="theme-color" content="#ffffff" />
+    <link rel="manifest" href="/manifest.webmanifest" />
```

Comment:
```suggestion
    <link rel="manifest" href="/manifest.json" />
```

Harusnya sih manifest jaman sekarang pakenya json, ga pake `.manifest` lagi. Sekalian juga diubah nama file yang ada di folder public, `manifest.webmanifest` jadi `manifest.json`

Reply:
Sekarang udah diganti bukan pake manifest.json tapi pake manifest.webmanifest

Reply:
Hmm, oke